app/Http/Routes/BaseRoutes.php

```diff
@@ -14,6 +14,9 @@ public function map(Router $router) {
             return redirect()->route('index');
         });
 
+        // Password Generation
+        $router->get('/password-gen/{length}', [ 'as' => 'admin.accounts.password', 'uses' => 'Base\IndexController@getPassword' ]);
```

Comment:
This shouldn't be named `admin.accounts.password` since it is a base route. I might suggest `password-gen`.